cli/src/main/kotlin/org/eclipse/apoapsis/ortserver/cli/LogsCommand/LogsCommand.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.cli
+
+import com.github.ajalt.clikt.command.SuspendingCliktCommand
+import com.github.ajalt.clikt.core.MutuallyExclusiveGroupException
+import com.github.ajalt.clikt.core.ProgramResult
+import com.github.ajalt.clikt.core.UsageError
+import com.github.ajalt.clikt.parameters.groups.cooccurring
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.options.split
+import com.github.ajalt.clikt.parameters.types.enum
+import com.github.ajalt.clikt.parameters.types.file
+import com.github.ajalt.clikt.parameters.types.long
+
+import io.ktor.util.cio.use
+import io.ktor.util.cio.writeChannel
+import io.ktor.utils.io.copyTo
+
+import org.eclipse.apoapsis.ortserver.client.OrtServerClient
+import org.eclipse.apoapsis.ortserver.model.LogLevel
+
+class LogsCommand(val config: OrtServerOptions) : SuspendingCliktCommand() {
+    private val runId by option(
+        "--run-id",
+        envvar = "ORT_RUN_ID",
+        help = "The ID of the ORT run."
+    ).long()
+
+    private val ortRunByIndex by OrtRunByIndexOptions().cooccurring()
+
+    private val outputDir by option(
+        "--output-dir",
+        "-o",
+        envvar = "ORT_DOWNLOAD_LOGS_OUTPUT_DIR",
+        help = "The directory to download the logs to."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    private val level by option(
+        "--level",
+        envvar = "ORT_DOWNLOAD_LOGS_LEVEL",
+        help = "The log level of the logs to download, one of ${LogLevel.entries.joinToString(", ")}."
+    ).enum<LogLevel>()
+
+    private val steps by option(
+        "--steps",
+        envvar = "ORT_DOWNLOAD_LOGS_STEPS",
+        help = "The run steps for which logs are to be retrieved, separated by commas."
+    ).split(",")
+        .default(emptyList())
```

Comment:
Similar to the download reports command, could we provide an enum or a predefined set of options for the user?